src/airdrop/src/utils.rs

```diff
@@ -50,7 +50,13 @@ pub fn add_user_to_airdrop_reward(
     eth_address: EthereumAddress,
     amount: AirdropAmount,
     reward_type: RewardType,
-) {
+) -> CustomResult<()>{
+
+    // check that we have enough tokens left
+    if state.total_tokens < amount.0 {
```

Comment:
NTH: would be nice to have an impl instead of always doing `amount.0`, like `amount::e8s()` or what ever value it contains